server/bleep/src/semantic.rs

```diff
@@ -178,33 +178,50 @@ impl Semantic {
             anyhow::bail!("no search target for query");
         };
 
-        let repo_filter: qdrant_client::qdrant::Condition = {
+        let repo_filter = {
             let conditions = parsed_query
                 .repos()
-                .map(|r| make_kv_filter("repo_name", r).into())
-                .collect();
+                .map(|r| {
+                    if r.contains('/') {
+                        format!("github.com/{r}")
```

Comment:
I would make this conditional, so we never end up with `github.com/github.com/org/repo` accidentally?